Infoblox provider support for DNS view

dsbrng25b opened the pull request:
Add flag  `--infoblox-view` for Infoblox provider to configure the DNS view as described in #831 . Unfortunately i don't have a Infoblox instance to test the changes. @dimatha can you test this?

njuettner commented:
Thank you for your PR, do you mind adding tests as well?

dsbrng25b commented:
I added the test for the new Infoblox flag.
Currently the build is failing because there seems to be a problem with coveralls.

dimatha commented:
Hello David, thank you very much for your assistance and support! I’m on sick leave this week and will run necessary checks next week once I’m back.

BR,
Dmitry

> On 19. Feb 2019, at 16:29, David Schneider <notifications@github.com> wrote:
> 
> I added the test for the new Infoblox flag.
> Currently the build is failing because there seems to be a problem with coveralls.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.

gregsidelinger commented:
I can confirm I was able to use this patch to get external-dns talking to the Infoblox at my company. Without the View support I could not create any records.

njuettner commented:
LGTM! WDYT @dsbrng25b can we merge it?

dsbrng25b commented:
Today I had the chance to test the new flag against a running Infoblox instance and it worked.
@njuettner  I rebased again and changed the parameter help text slightly so the PR can now be merged.
Cheers

njuettner commented:
/lgtm
/approve
